src/System.Net.Sockets/src/System/Net/Sockets/SocketInformation.cs

```diff
@@ -2,138 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-#if !netcore50
-using System.Runtime.Serialization;
-#endif
+using System.Runtime.InteropServices;
 
 namespace System.Net.Sockets
 {
 #if !netcore50
```

Comment:
I removed the other one because the using was no longer needed.  I don't know about this one; if I can remove it and the BuildAllConfigurations still passes, then I will.